src/_P081_Cron.ino

```diff
@@ -77,7 +77,7 @@ private:
 union timeToFloat
 {
   time_t time;
```

Comment:
Maybe add an assert here to make sure `time_t` is 32 bit?
This way we know at compile time whether it may cause an issue or not.

Reply:
yes and I think it needs a bit more testing, I'm not 100% sure that this is a good solution (just setting the high byte to 0x00), not sure about side effects...